vignettes/decorate-module-output.Rmd

```diff
@@ -510,7 +515,7 @@ interactive_decorator_3 <- teal_transform_module(
 
 ### Application
 
-As you might have noted, the x axis title from the first decorator will be used but won't show up on the resulting plot:
+As you might have noticed, the x-axis title from the first decorator will be used but will not appear in the resulting plot:
```

Comment:
@gogonzo 
Can you double check the app for this section? From what I can see, only `decorator_1` is functional.
No changes to the plot's x or y axis when I toggled to `decorator_2` and `decorator_3`. 

Is this as expected?

Reply:
Yes, third decorator doesn't make sense.

I think this vignette is too long. There are too many examples doing every possible thing. I think that instead of having a section with multiple decorators there should be a tab in the first example with two decorators. Also the last example with multiple outputs - Yes, it is a real case but come on, let's give app developers space for creativity and let them utilise decorators in their way.

Reply:
>  I think that instead of having a section with multiple decorators there should be a tab in the first example with two decorators.

I like this idea.

> Also the last example with multiple outputs - Yes, it is a real case but come on, let's give app developers space for creativity and let them utilise decorators in their way.

Now that you pointed out, it does feel repetitive with the section `Decorating Plots`. Maybe the section `Decorating Plots` should be named something that's more generic that can easily be applied to either plot, table, or whatever the final output of the module.

Let me try to rewrite this.

Reply:
Update in e97a7da97f3c577acb7b925b284637a75b20be22